packages/client/src/App.jsx

```diff
@@ -1,10 +1,11 @@
 import React, { Component } from 'react'
 import { Provider } from 'react-redux'
+import { MuiThemeProvider as ThemeProvider } from '@material-ui/core/styles'
+import theme from '@serge/components/themes/serge'
```

Comment:
Ooh, this confused me :-D

I was looking `packages/theme` for this content.  But, I now realise it's in `packages/components/themes`.

I'm happy to go with your guidance, but is the `themes` package the better place to keep themes?

(I don't mind what the answer is, I'm just inquisitive)

Reply:
I was planning to put that into themes, but there are major differences between the two theme definitions. While the existing @serge/themes contains scss based files, the @serge/components/themes is simply a .js file that derived its configuration from material-ui theme. 

Moving it into the @serge/themes should be feasible if we're going to that direction.

Reply:
Thanks for that @zukilover .

Do you have an opinion/preference/suggestion on moving to @serge/themes ?

Reply:
I think we can do this later on #498 so that we can bo both theme migration and integration with styled system at the same time.